test/key-value-model.test.js

```diff
@@ -138,16 +139,16 @@ describe('KeyValueModel', function() {
   });
 
   function setupAppAndCacheItem() {
-    app = loopback({ localRegistry: true, loadBuiltinModels: true });
+    app = loopback({localRegistry: true, loadBuiltinModels: true});
     app.use(loopback.rest());
 
     CacheItem = app.registry.createModel({
       name: 'CacheItem',
       base: 'KeyValueModel',
     });
 
-    app.dataSource('kv', { connector: 'kv-memory' });
-    app.model(CacheItem, { dataSource: 'kv' });
+    app.dataSource('kv', {adapter: 'kv-memory'});
```

Comment:
I would get the following error:
```
  3) KeyValueModel "before each" hook: setupAppAndCacheItem for "provides "get(key)" at "GET /key"":
     TypeError: Cannot create data source "kv": Cannot create property 'name' on string 'kv-memory'
    at dataSourcesFromConfig (lib/application.js:427:29)
    at EventEmitter.app.dataSource (lib/application.js:226:14)
    at Context.setupAppAndCacheItem (test/key-value-model.test.js:150:9)
```

I did some research and it seems to be `adapter` is the equivalent and it passes